src/api/github-repo/services/github-repo.js

```diff
@@ -120,6 +126,85 @@ class GithubManager {
     hasExpired(cacheEntry) {
         return cacheEntry.addedTime + CACHE_TTL < Date.now();
     }
+
+    async getInstallationToken() {
+        const now = Math.floor(Date.now() / 1000);
+
+        if (privateKey.startsWith('LS0t') || !privateKey.startsWith('-----BEGIN')) {
+            // Looks base64-encoded
+            privateKey = Buffer.from(privateKey, 'base64').toString('utf8');
+        }
+
+        const token = jwt.sign(
+            {
+                iat: now - 60,
```

Comment:
Can we make these magic number to constants? That will also help us understand the unit.